### ISSUE TITLE
Sync field will not be calculated

### ISSUE THREAD

**Issue by YujiShen** [CalderaWP/Caldera-Forms#3162]
## What Version Of Caldera Forms, WordPress and PHP Are You Using?
1.8.4

## What Is The Unexpected Behaviour?
I set up a number field A to sync with another number field B as default, and then use that synced field A as variable in a calculation field. But the calculation will not work only after I manually changed the synced field A, even though there is number in field A.

**Comment by New0** [comment]
Hello @YujiShen , this could be related to calculation values bindings, and if so a possible duplicate of #3146 

Could you test using this build of Caldera Forms ? 
[1.8.4.3146.zip](https://github.com/CalderaWP/Caldera-Forms/files/3127129/1.8.4.3146.zip)

**Comment by Shelob9** [comment]
@New0 Fixed via  via #3146 and dca17bb ??
